deploy/examples/common-external.yaml

```diff
@@ -18,11 +18,6 @@
 #
 # Then kubectl create -f cluster-external.yaml
 ###################################################################################################################
-apiVersion: v1
```

Comment:
where is the other place we create?

Reply:
https://github.com/rook/rook/blob/release-1.13/deploy/examples/import-external-cluster.sh#L71